packages/form-js-viewer/test/spec/render/components/form-fields/Number.spec.js

```diff
@@ -94,26 +94,53 @@ describe('Number', function() {
   });
 
 
-  it('should handle change', function() {
+  describe('change handling', function() {
```

Comment:
I believe we could further improve the field specs (clearly structure according to `render`, `change`, ...).

But it is fine to do that one day in the future :).